src/state/KindeProvider.tsx

```diff
@@ -10,6 +10,7 @@ import { initialState } from './initialState';
 import { KindeContext } from './KindeContext';
 import { reducer } from './reducer';
 import { version } from '../utils/version';
+import { KindeUser, State } from './types';
```

Comment:
@stephenkao sorry one last minor thing, but now that we're using the `object` type, we don't need to import `State` here any more :)

Reply:
Not a problem -- thanks for catching that!  Just pushed up [a commit](https://github.com/kinde-oss/kinde-auth-react/pull/26/commits/42dbed83879e7ce97367e107c2648ce06a78965d) that removes it :)

Reply:
Champion, thanks!